coil-svg/src/commonMain/kotlin/coil3/decode/SvgDecoder.kt

```diff
@@ -0,0 +1,24 @@
+package coil3.decode
+
+import coil3.request.Options
+
+/**
+ * A [Decoder] that uses [AndroidSVG](https://bigbadaboom.github.io/androidsvg/) and  to decode SVG
```

Comment:
This should be swapped with the Android implementation docs

Reply:
Done